README.md

```diff
@@ -38,6 +38,23 @@ GET /report-and-manage-your-allowance-for-customs-duty-waiver-claims
 ```
 This will prompt the user to log in if there isn't an active session yet.
 
+## Changing the Sector Cap
+
+The sector cap configuration is stored in the **application.conf** file. Navigate to the file and locate the sector cap config below:
```

Comment:
Instead of `The sector cap configuration is stored in the **application.conf** file. Navigate to the file and locate the sector cap config below:` can we have something like: `Industry sector caps are likely to be updated from time to time. To avoid needing a new release each time an update is required, the sector cap is configured in **application.conf**:`

Reply:
feel free to improve what i wrote